VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimTimeline/ClaimTimeline.tsx

```diff
@@ -35,12 +36,12 @@ function ClaimTimeline({ attributes, claimID }: ClaimTimelineProps) {
 
   return (
     <Box>
-      {itemsNeededFromVet && !attributes.waiverSubmitted && (
+      {itemsNeededFromVet && !attributes.waiverSubmitted && !featureEnabled('claimPhaseExpansion') && (
```

Comment:
I don't love this code. This is hard to follow logically. Essentially we are...changing nothing on the screen here but updating some content string values, but also removing this alertbox in the version of the code where the feature flag is on. We should have a deprecatedClaimTimeline file here that we navigate to when the feature flag is off, and the copy shouldn't have this alertbox.

Reply:
updated